doc/user-guide.adoc

```diff
@@ -29,6 +29,46 @@ The `applicationImage` accepts different references to Docker image:
 * a tag: `quay.io/wildfly-quickstarts/wildfly-operator-quickstart:18.0`
 * a digest: `quay.io/wildfly-quickstarts/wildfly-operator-quickstart@sha256:0af38bc38be93116b6a1d86a9c78bd14cd527121970899d719baf78e5dc7bfd2`
 
+[[source-repository]]
+## Specify the Application Source
+
+THe WildFly Operator can also take as input the source code of the application and use https://github.com/openshift/source-to-image[OpenShift Source-to-Image (S2I)] to build the Java applications and create a Docker image from it.
+
+[IMPORTANT]
+====
+This feature is available on OpenShift only.
+
+It relies on OpenShift-specific resources such as `BuildConfig` and `ImageStreams` that are not available on a plain Kubernetes cluster.
+====
+
+[source,yaml]
+.Example of application source configuration
+----
+spec:
+  applicationSource:
+    sourceRepository:
+      url: "https://github.com/jmesnil/wildfly-operator-quickstart.git"
```

Comment:
is using your fork here intentional for final implementation or is it only for development and it should be fixed before merging?

Reply:
fixed